.github/dependabot.yml

```
@@ -0,0 +1,25 @@
+version: 2
+updates:
+  # Check for updates on our docker image and for celestia-node
+  - package-ecosystem: docker
+    directory: "/ops-bedrock"
+    schedule:
+      interval: daily
+    open-pull-requests-limit: 10
```

Should we add infra reviewers here?
```suggestion
    open-pull-requests-limit: 10
    reviewers
      - "ethereum-optimism/infra-reviewers"
```